.travis.yml

```diff
@@ -16,7 +16,7 @@ git:
 
 before_install:
   - pip install "setuptools>=18.5"
-  - sudo apt-get install -y postgresql-9.3-postgis-2.3 postgis
+  - sudo apt-get install -y postgresql-9.3-postgis-2.3
```

Comment:
It looks like `postgis` was never necessary, and now installing it brings in Postgres 10, which runs instead of the Postgres 9 we need.

Reply:
It's not necessary because `postgresql-9.3-postgis-2.3` already pulls in the right version?

Reply:
Yes, as far as I can tell that's right.